i18n.php

```diff
@@ -117,7 +117,7 @@
     'name' => 'Azerbaijani',
     'native' => 'Azərbaycanca',
     'rule' => 'Rule',
-    'scenario' => 'Nümunələr|Ssenari',
+    'scenario' => 'Nümunə|Ssenari',
```

Comment:
This should not be edited directly - we get translations from cucumber upstream

Reply:
Can you point me to where that is so I can make an upstream PR.

Reply:
(short reply was because I was on mobile)

https://github.com/cucumber/cucumber/blob/master/gherkin/gherkin-languages.json

After it's merged you can run `i18n.php` in this repo and make a PR from the result 👍

Reply:
See cucumber PR https://github.com/cucumber/cucumber/pull/559